README.md

```diff
@@ -35,22 +35,94 @@ CloneKit::Specification.new(BlogPost) do |spec|
         CloneKit::Rules::Remap.new("BlogPost", "Account" => "account_id", "BlogType" => "blog_type_id")
       ]
     )
+    spec.after_operation do |operation|
+      ...
+    end
 end
 ```
 
 ## Writing an Emitter
 
-You have to write some emitters for your app. By default, CloneKit specifications utilize and empty emitter, making all clones no-operations.
+By default, CloneKit specifications utilize an empty emitter, making all clones no-ops. Emitters are expected to make db calls using logic defined in the emitter.
 
-TODO
+#### Emitter rules
+  - Emitters must respond to `#emit_all` and `#scope`.
+  - `emit_all` must return an object that responds to `#pluck`.
 
-## Writing a Cloner
+```ruby
+CloneKit::MongoSpecification.new(BlogPost) do |spec|
+  ...
+  spec.emitter = ActiveRecordEmitter.new(BlogPost)
```

Comment:
:neckbeard: I would probably expect something wrapped in `MongoSpecification` to not use something called `ActiveRecordEmitter`.  It might be more clear if these both reference Mongo or AR.

Reply:
Good point. Updated both.